pkg/tools/sandbox/builder.go

```diff
@@ -360,6 +409,16 @@ func (b *Builder) SetupRegistryClients(ctx context.Context, node *Node) {
 	node.NSRegistryClient = client.NewNetworkServiceRegistryClient(nsmgrCC)
 }
 
+// newAddress - will return a new public address, if unixSockets are used prefix will be used to make uniq files.
+func (b *Builder) newAddress(prefix string) string {
+	if !b.useUnixSockets {
+		return "127.0.0.1:0"
+	}
+
+	b.addressID++
+	return fmt.Sprintf("unix:%s/%s_%d.sock", b.sockPath, prefix, b.addressID)
```

Comment:
Can we use uuid instead of `b.addressID`?

Reply:
Sure :-) Fixed.